server/routerlicious/packages/services-shared/src/socketIoServer.ts

```diff
@@ -119,7 +119,8 @@ export function create(
     const io = new Server(server, {
         // enable compatibility with socket.io v2 clients
         allowEIO3: true,
-        perMessageDeflate: false,
+        // Indicates whether a connection should use compression
+        perMessageDeflate: true,
```

Comment:
Is it possible that this setting can cause a spike in memory as we suspected last time we turned it off? If yes, do we want to make it configurable through config.json, so it's easier to disable that if needed during an emergency?

Reply:
I have the same concern. There can be a CPU and memory spike (https://github.com/gorilla/websocket/issues/203).

@GaryWilber Does push enable it?

Reply:
Yes, I saw Henrique's comment after merging the code. Will add the config as he suggested. Push is not enabling it because AFD does not support this extension. With respect to the memory and cpu consumption, pasting the offline conversation I had with Henrique:

> It is a valid  point but I do not think we will get affected by it for a few reasons:
>
> The theory that this flag was causing the memory issues is probably not true. The default for this flag was always 
 false..I just added this line explicitly to make doubly sure.
>
> This comes into play only for socket payloads that are more than 1mb..that that does not happen in most cases for us
>
>I stress tested it pretty extensively and did not see any difference in the memory or cpu usage